ci-operator/step-registry/baremetal/lab/agent/day2/baremetal-lab-agent-day2-commands.sh

```diff
@@ -99,16 +104,16 @@ cp "${SHARED_DIR}/nodes-config.yaml" "${DAY2_INSTALL_DIR}/"
 cp "${SHARED_DIR}/nodes-config.yaml" "${ARTIFACT_DIR}/"
 
 echo "Create node.iso for day2 worker nodes..."
-/cli/oc adm node-image create --dir="${DAY2_INSTALL_DIR}" -a "${day2_pull_secret}" --insecure=true
+/tmp/oc adm node-image create --dir="${DAY2_INSTALL_DIR}" -a "${day2_pull_secret}" --insecure=true
```

Comment:
Hi @zniu1011, is the `--insecure=true` really needed here? could be possible to make it work with HTTPS or do we have some impediment?

Reply:
Hi @pamoedom , it is needed here, which is used for disconnected case, to pass the self-signed certificate for local mirror registry. I didn't know other solutions here for now in this case.

Reply:
got it, it's a disconnected case, thks for the confirmation @zniu1011